lemur/common/celery.py

```diff
@@ -273,7 +274,8 @@ def fetch_acme_cert(id):
         real_cert = cert.get("cert")
         # It's necessary to reload the pending cert due to detached instance: http://sqlalche.me/e/bhk3
         pending_cert = pending_certificate_service.get(cert.get("pending_cert").id)
-        if not pending_cert:
+        if not pending_cert or pending_cert.resolved:
```

Comment:
I am hoping `pending_cert.resolved` would address the issue of 1 duplicate certificate. 

This seems to be a corner case, since currently cert issuance triggers `fetch_acme_cert ` and also celery's schedule. If they happen to be both within the same time range, celery wont notice that the pending certificate is resolved.